h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -64,6 +64,8 @@
   public String[][] _gamColnames = null;
   public int[][] _gamColIndices = null; // corresponding column indices in dataInfo
   public static int _totalBetaLen;
+  private boolean _calScoreKeeper = false; // true to update scored_train/scored_valid/scored_xval for early stop
```

Comment:
should this be just _earlyStoppingEnabled? the commend does not make what the name means

Reply:
thanks for the name